runtime/frequency/src/lib.rs

```diff
@@ -222,7 +218,7 @@ pub type Executive = frame_executive::Executive<
 	AllPalletsWithSystem,
 	(
 		pallet_messages::migration::v2::MigrateToV2<Runtime>,
-		pallet_schemas::migration::v2::MigrateToV2<Runtime>,
+		pallet_schemas::migration::v3::MigrateToV3<Runtime>,
```

Comment:
Should we delete the v2 rust file?

Reply:
It depends! I didn't remove it since it documents the historical data changes and since it's not part of the code it would not be included in the wasm but I think there are benefits for having it part of the codebase

Reply:
Why is the migrate to v2 line still in, has it not been run on mainnet yet?